updating docs associated to release and github workflow process (#1681)
Kong/kubernetes-ingress-controller: README.md

for Kubernetes Services, all using
Custom Resource Definitions(CRDs) and Kubernetes-native tooling.

> [**Features**](#features) | [**Get started**](#get-started) | [**Documentation**](#documentation) | [**main branch builds**](#mainnext-branch-builds) | [**Seeking help**](#seeking-help)

Reviewer: ```suggestion
[**Features**](#features) | [**Get started**](#get-started) | [**Documentation**](#documentation) | [**main branch builds**](#main-branch-builds) | [**Seeking help**](#seeking-help)
```

Please browse through [guides][guides] to get started or understand how to configure
a specific setting with Kong Ingress Controller.

## main branch builds

> Pre-release builds of the `main` branches are available from the

Reviewer: ```suggestion
Pre-release builds of the `main` branch are available from the
```

[kong-ingress-controller repository][docker-images] hosted on Docker Hub:

> `main` contains unreleased patch code for the current release:

Reviewer: ```suggestion
`main` contains unreleased new features for upcoming minor and major releases:
```
